src/styles/nnfx-dark.css

```diff
@@ -11,7 +11,8 @@
   color: #fff;
 }
 
-.xml .hljs-meta {
+.language-xml .hljs-meta,
+.language-xml .hljs-meta-string {
```

Comment:
Ah, I should go back and look for this in other themes... I thought I remembered taking care of it but obviously I missed some. Thanks!